main.go

```diff
@@ -1,7 +1,65 @@
 package main
 
-import "fmt"
+import (
+	"fmt"
+	"time"
+)
+
+type Item struct {
+	title  string
+	author string
+	score  int
+	time   time.Time
+	URL    string
+}
+
+type Items struct {
+	Item []struct {
+	}
+}
```

Comment:
Still no, this is what you want:

```suggestion
type Items []Item
```

Make sure you understand it :)